Add support for KVM flag (Managed Identity)

lmoelleb opened the pull request:
Sets the KVM flag (Azure Managed Identity) if the following environment variable is set:
DUCTTAPE_SIGN_AZ_KEY_VAULT_MANAGED_IDENTITY=true

Commit:
Add support for KVM flag (Managed Identity)

Commit:
Updated readme.md with new kvm command line option.

kung-foo commented:
does this require a newer build of az sign tool (I see you made a recent PR to that repo)?

lmoelleb commented:
> does this require a newer build of az sign tool (I see you made a recent PR to that repo)?

Yes. This change works with the main branch of AzureSignTool - not with the current release.

kung-foo commented:
OK, then the documentation should reflect that using that flag requires at a build from at least https://github.com/vcsjones/AzureSignTool/commit/028275203bfcd87ec5db335c0d16f66929bf7275

Commit:
Document --azure-key-vault-managed-identity requires new build of AzureSignTool.

lmoelleb commented:
> OK, then the documentation should reflect that using that flag requires at a build from at least [vcsjones/AzureSignTool@0282752](https://github.com/vcsjones/AzureSignTool/commit/028275203bfcd87ec5db335c0d16f66929bf7275)

Added to the displayed help text as well as Readme.md. I was not sure of the best way to add it in Readme.md, as I would like to include the link. In the end I added it after the preformatted section. If you prefer it some other place, please let me know.

kung-foo commented:
released here: https://github.com/Intelecy/ducttape/releases/tag/v0.0.6

Thanks for the PR!

lmoelleb commented:
Great, thanks – saves me installing Go on the build machines - And I can now put Go on my CV. :D

I just hope AzureSignTool is updated soon, the current version is a disaster (does not output error messages).
